ably/realtime_conn_spec_integration_test.go

```diff
@@ -2731,9 +2657,19 @@ func TestRealtimeConn_RTN19b(t *testing.T) {
 			Action:  ably.ActionDetach,
 		},
 	}
-	for _, expect := range msgs {
+
+	gots = nil
+	for range msgs {
 		var got *ably.ProtocolMessage
 		ablytest.Instantly.Recv(t, &got, out, t.Fatalf)
+		gots = append(gots, got)
+	}
+
+	sort.Slice(msgs, func(i, j int) bool { return  msgs[i].Action < msgs[j].Action})
```

Comment:
```suggestion

```
Again I don't think we need to sort something thats already hardcoded in a sorted order.